repo/packages/K/kubernetes/1/marathon.json

```diff
@@ -2,6 +2,7 @@
   "id": "{{kubernetes.framework-name}}",
   "cpus": {{kubernetes.cpus}},
   "mem": {{kubernetes.mem}},
+  "acceptedResourceRoles": {{kubernetes.resource-roles}},
```

Comment:
Yes it is. There's rules published somewhere about this.

On Wed, Sep 30, 2015 at 5:54 PM, Karl Isenberg <notifications@github.com>
wrote:

> In repo/packages/K/kubernetes/1/marathon.json
> <https://github.com/mesosphere/multiverse/pull/13#discussion_r40857485>:
>
> > @@ -2,6 +2,7 @@
> >    "id": "{{kubernetes.framework-name}}",
> >    "cpus": {{kubernetes.cpus}},
> >    "mem": {{kubernetes.mem}},
> > +  "acceptedResourceRoles": {{kubernetes.resource-roles}},
>
> If it has a default is it really required?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/multiverse/pull/13/files#r40857485>.
>